From fbefdcfc31b00ae5ff439ad2e57608da9d0fa9d2 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:22:30 -0400
Subject: [redhat] kabi: Adding symbol bio_add_page (fs/bio.c)

Message-id: <071cef4e969783fefad42c30277103d1b21cd0af.1410510773.git.poros@redhat.com>
Patchwork-id: 95969
O-Subject: [PATCH RHEL7.1 030/233] kabi: Adding symbol bio_add_page (fs/bio.c)
Bugzilla: 1071098
RH-Acked-by: Jeff Moyer <jmoyer@redhat.com>

Bugzilla: 1071098
    https://bugzilla.redhat.com/show_bug.cgi?id=1071098

    Description:
    ============
    Attempt to add page to bio.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/bio_add_page
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/bio_add_page
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/bio_add_page

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/bio_add_page  b/redhat/kabi/kabi-module/kabi_ppc64/bio_add_page 
new file mode 100644
index 0000000..0f35244
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/bio_add_page  
@@ -0,0 +1,2 @@
+#1-
+0xbc897608 bio_add_page vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/bio_add_page  b/redhat/kabi/kabi-module/kabi_s390x/bio_add_page 
new file mode 100644
index 0000000..020ed56
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/bio_add_page  
@@ -0,0 +1,2 @@
+#1-
+0x6dbe029f bio_add_page vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/bio_add_page  b/redhat/kabi/kabi-module/kabi_x86_64/bio_add_page 
new file mode 100644
index 0000000..ba04c5b
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/bio_add_page  
@@ -0,0 +1,2 @@
+#1-
+0x4554b079 bio_add_page vmlinux EXPORT_SYMBOL
-- 
1.7.1